google/cloud/firestore_v1/base_query.py

```diff
@@ -651,7 +788,7 @@ def end_at(
             document_fields_or_snapshot, before=False, start=False
         )
 
-    def _filters_pb(self) -> StructuredQuery.Filter:
+    def _filters_pb(self) -> Optional[StructuredQuery.Filter]:
```

Comment:
The `Optional` is a type annotation in Python, and it's more of a documentation change than behavioral change.
The `Optional[sometype]` means that it can either be `sometype` or `None`.
This function was already returning `None` before (in line 666 below)
```
    if num_filters == 0:
        return None
```
So this is not a change in the return value. It was actually a bug that the return value was not typed as `Optional`. This change fixed the bug and make it accurate.